tlsr9/ble/stack/ble/TL321X/ble.h

```diff
@@ -1,33 +1,64 @@
-/******************************************************************************
- * Copyright (c) 2023 Telink Semiconductor (Shanghai) Co., Ltd. ("TELINK")
- * All rights reserved.
+/********************************************************************************************************
+ * @file    ble.h
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * @brief   This is the header file for BLE SDK
  *
- *   http://www.apache.org/licenses/LICENSE-2.0
+ * @author  BLE GROUP
+ * @date    06,2022
  *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * @par     Copyright (c) 2022, Telink Semiconductor (Shanghai) Co., Ltd. ("TELINK")
  *
- *****************************************************************************/
+ *          Licensed under the Apache License, Version 2.0 (the "License");
+ *          you may not use this file except in compliance with the License.
+ *          You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *          Unless required by applicable law or agreed to in writing, software
+ *          distributed under the License is distributed on an "AS IS" BASIS,
+ *          WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *          See the License for the specific language governing permissions and
+ *          limitations under the License.
+ *
+ *******************************************************************************************************/
 #ifndef BLE_H_
 #define BLE_H_
 
 
-#include "ble_config.h"
+
 #include "ble_common.h"
 #include "ble_format.h"
 
 #include "controller/ble_controller.h"
+// #include "host/ble_host.h"
+// #include "profile/ble_profile.h"
 
 #include "hci/hci.h"
 #include "hci/hci_const.h"
 #include "hci/hci_cmd.h"
 #include "hci/hci_event.h"
 
+// #include "service/ota/ota.h"
+// #include "service/ota/ota_server.h"
+// #include "service/device_information.h"
```

Comment:
To mach commented (unused) code
Please remove all commented code from all TLX files

Reply:
will delete them.